kasa/smart/modules/lightstripeffect.py

```diff
@@ -0,0 +1,109 @@
+"""Module for light effects."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ...interfaces.lighteffect import LightEffect as LightEffectInterface
+from ..effects import EFFECT_MAPPING, EFFECT_NAMES
+from ..smartmodule import SmartModule
+
+if TYPE_CHECKING:
+    pass
```

Comment:
```suggestion
```

Reply:
Removed.  I can't quire work out how these keep appearing as I don't recall writing it.  Maybe ruff is removing a redundant type checking import